src/main/java/uk/gov/hmcts/juror/job/execution/jobs/dashboard/ams/data/Certificates.java

```diff
@@ -82,13 +81,7 @@ public Job.Result populate() {
                 expiryDate,
                 status
             );
-            if (expiryDate != null && expiryDate.after(DateUtils.addDays(Date.from(clock.instant()), -30))
-                && config.getSmtp() != null) {
-                smtpService.sendEmail(config.getSmtp(),
-                    "PNC Certificate Expires soon",
-                    "The PNC Certificate is due to expire at " + expiryDate + ". Please update the certificate.",
-                    config.getEmailRecipients());
-            }
+
             populateTimestamp(dashboardData, "Certificates", LocalDateTime.now(clock));
```

Comment:
This Job can be deleted as we can no longer check expiry

Reply:
The expiry job is still needed for the dashboard stats. so the idea would be to copy the cert from the PNC server into the dashboard to get the stat using the job.